de-lib/src/main/java/org/iplantc/de/client/util/CopyToClipboardUtil.java

```diff
@@ -0,0 +1,58 @@
+package org.iplantc.de.client.util;
+
+/**
+ * A util class to interact with js copy and clipboard functionalities
+ * 
+ * Created by sriram on 11/2/17.
+ */
+public class CopyToClipboardUtil {
+
+    /**
+     * Copy text value from the textfield to clipboard
+     *
+     * @param id id of the textfield
+     * @return boolean whether copy succeeded or failed.
+     */
+    public static native boolean copyToClipboard(String id) /*-{
+        var copyText = $doc.getElementById(id);
+        copyText.select();
```

Comment:
We may as well also move this inside the try/catch block, in case a bad ID is passed in?